config/crd/bases/iam.cnrm.cloud.google.com_iamauditconfigs.yaml

```diff
@@ -0,0 +1,168 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I'm also surprised by these 4 IAM CRDs, I wouldn't expect to see net new CRDs here.

Reply:
The placement is also weird. I woouldn't expect them under bases. Trying to remove config/crd/bases and see if make ready-pr regens them.

Reply:
make ready-pr did not add them back so they should be gone.